pkg/source/elasticsearch/persistence.go

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
Is it possible to store the offset directly to elasticsearch?

Reply:
If the Loggie Pod is migrated, the local records in sqlite will be lost.

Reply:
> If the Loggie Pod is migrated, the local records in sqlite will be lost.

Yes, it does get lost.

Reply:
> If the Loggie Pod is migrated, the local records in sqlite will be lost.

But I'm curious if file(source) has the same problem？

Reply:
- The collected files will only be in the node and they never migrated, so the file source would not have this problem.
- Elasticsearch source should deployed independently based on the form of aggregator, the probability of Pod migration is relatively high, so there will be problems when recording in local sqlite.
- In fact, it can be same as kafka source, and kafka source will record the offset into a topic in kafka, so the elasticsearch source here follows a similar principle, but the elasticsearch source needs to implement a offset mechanism by itself.